<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="sony-VAIO" ignored="0" name="restassuredjartest" tests="5" failures="1" timestamp="2020-04-26T22:24:40 IST" time="16.982" errors="0">
  <testcase name="getUserDetails" time="5.042" classname="testCases.GetRespBody"/>
  <testcase name="getUserDetails" time="2.784" classname="testCases.GetStatcodeStatline"/>
  <testcase name="user" time="3.794" classname="testCases.GetUseofJSONpath"/>
  <testcase name="getUserDetails" time="2.897" classname="testCases.GetResponseFromDirectURL"/>
  <testcase name="deleteEmpRecord" time="1.626" classname="testCases.DeleteTC">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.DeleteTC.deleteEmpRecord(DeleteTC.java:28)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteEmpRecord -->
</testsuite> <!-- restassuredjartest -->
